### PR TITLE
If multiple countries allowed, phone number validation performed against first country only.

### PR DESCRIPTION
**Pull request by kburdon**
There is a bug in the phone validation for multiple countries.  When multiple countries are allowed, the phone number is validated against the first country and returns invalid if the phone number is not valid for the first country being tested.  The other countries are never tested.

There is a catch-all return false at the end of the loop, which should execute if all country validations fail.  I updated the logic so that the validation returns true if the phone number validates successfully against the current country in the loop.  If not, the loop continues to execute until either the phone number validates successfully against a country in the array, or all countries have been exhausted, in which case the fall-back existing return false will return.

The unit tests have also been updated to test this scenario; the exiting unit test to test for multiple countries passes because the country the test number is valid for happens to be the first country in the array.

### CODE REVIEW

**Comment by Propaganistas** [comment]
Great catch! And thanks for the top-notch PR.

I'll roll out a new release in a second.

**Comment by kburdon** [comment]
Just wanted to leave a little note in here about this for future reference.  Now that the code is looping through all countries, if Austria is included, it would appear that any number is going to come back as valid.  Here's why:

https://en.wikipedia.org/wiki/Telephone_numbers_in_Austria

"There are no standard lengths for either area codes or subscribers' numbers in Austria, meaning that some subscribers' numbers may be as short as three digits. Larger towns have shorter area codes permitting longer subscriber numbers in that area."

Basically it appears that Austrian phone numbers can be any number of digits.  So it's not a bug if a seemingly invalid phone number returns valid if Austria is in the allowed countries list, it's just Austria being Austria :)
